docs/guide/topics/webhooks.rst

```diff
@@ -0,0 +1,219 @@
+.. currentmodule:: discord
+
+.. _guide_webhooks:
+
+Webhooks
+========
+
+Webhooks are used for posting messages in Discord text channels without relying on a bot user.
+
+The focus of this section is to provide a basic understanding of webhooks and
+working with them using discord.py.
+
+For detailed information about webhooks, See the documentation for :ref:`webhooks <discord-api-webhooks>`.
+
+Introduction to webhooks
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+The difference between sending messages with webhooks and bots is that bots require proper authentication and a bot user
+whereas webhooks don't need either.
+
+There are currently three types of webhooks:
+
+1. Incoming webhooks
+2. Channel follower webhooks
+3. Application webhooks
+
+**Incoming webhooks** are associated to a channel and can post messages in that channel
+using a webhook token.
+
+**Channel follower webhooks** are used for crossposting of messages in news channels
+by Discord. This type does not have any token associated to it and bots cannot post
+messages through these webhooks.
+
+**Application webhooks** are used for creating interaction responses. The management of
+this type of webhooks are handled internally by the library and is not in scope of this guide.
+
+This guide will primarily focus on the first type i.e Incoming webhooks.
+
+Creating a webhook
+~~~~~~~~~~~~~~~~~~
+
+In order to create incoming webhooks in a channel, the member needs the
+:attr:`~Permissions.manage_webhooks` permission in the given guild or text channel.
+
+To create a webhook, Navigate to "Server Settings > Integrations" page.
```

Comment:
I feel whilst this is 100% correct, it should also mention that you can create webhooks on a channel too.
The "little cog" on the channel -> integrations -> create webhook

Reply:
Thanks, I have mentioned that. Let me know if there can be any improvement in the wording.